Topics/Tech_Stacks/Bootstrap.md

```diff
@@ -0,0 +1,163 @@
+# Learning Bootstrap, a CSS Framework
+
+## Table of contents
+### [Introduction](#what-is-bootstrap?-1)
+### [What is Bootstrap](#what-is-bootstrap-1)
+### [Where to begin](#where-to-begin-1)
+### [Bootstrap Basics](#bootstrap-basics-1)
+### [References](#references-1)
+
+## Preface
+Here you will learn the basics of how to incorporate Bootstrap into your website-building workflow. This should not serve as an in-depth guide, but rather one that should help you get your foot in the door to begin understanding and using Bootstrap components. Before you start, it is recommended that you have working knowledge of both HTML and CSS. This could be as little as a simple styled website. Although not required, it will be helpful in understanding why Bootstrap is such a widely-used and useful framework when working with CSS. At the end of this guide, you should be able to apply some simple Bootstrap components or formatting into your website.
```

Comment:
Here, one of the thing I suggest and think good idea that you add some tutorials about CSS and HTML. This does not have to be your tutorial, it can be outer source that teaches you some basic knowledge about CSS and HTML. Moreover, you can briefly explain why some basic knowledge about CSS and HTML are needed or recommended

Reply:
That is a good point. Since there is already a section in the wiki for CSS, I will make sure to link that section. On that note, I think I will also link the Bootstrap section in the CSS as a next step.